docs/content/getting-started.md

```diff
@@ -51,3 +51,27 @@ export default const MyApp = () => (
 ```
 
 This will apply the same `color`, `font-family`, and `line-height` styles to the `<body>` as [Primer CSS's base styles](https://github.com/primer/css/blob/master/src/base/base.scss#L15-L20).
+
+## TypeScript
+
+Primer Components includes TypeScript support with no additional `@types` package necessary. You will need to install `@types/styled-components` and `@types/react` if you import either of those packages.
+
+### Fixing "Duplicate identifier 'FormData'"
+
+Since version 14.1.9, `@types/styled-components` has a dependency of both `@types/react` and `@types/react-native`. Unfortunately, those declarations clash; for more information, see [issue 33311](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33311) and [issue 33015](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33015) in the DefinitelyTyped repo.
```

Comment:
Is this version 14.1.9 of styled-components or of Primer Components?

Reply:
Updated